### PR TITLE
96001 EPS JWT Token Wrapper

### PR DESCRIPTION
**Pull request by randomsync**
## Summary
This PR adds the JWT Wrapper class to get a signed assertion to call auth endpoint for EPS.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/96001

## Testing done

- [x] rspecs

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -1839,3 +1839,14 @@ schema_contract:
...
+vaos:
+  eps:
+    key_path: /fake/client/key/path
+    mock: false
+    client_id: "fake_client_id"
+    kid: "fake_kid"
+    audience_claim_url: "https://login.wellhive.com/oauth2/default/v1/token"
+    access_token_url: "https://login.wellhive.com/oauth2/default/v1/token"
+    api_url: "https://api.wellhive.com/care-navigation/v1"
+    scopes: "care-nav"
```
Comment (rmtolmach) [inline config/settings.yml:1852]: Do you plan to add these settings to the manifest repo? Instructions [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-VetsAPIsettingsandsecrets) if you need them.

Comment (randomsync) [reply config/settings.yml:1852]: yes, we have a separate ticket to add all configs and keys to manifest repo

Comment (rmtolmach) [reply config/settings.yml:1852]: ok, sounds good. This PR looks good from my perspective, so @ me when you get a teammate review and I'll review.

**Review by kanchanasuriya — APPROVED** [review]
LGTM

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**